beancount/utils/misc_utils.py

```diff
@@ -281,15 +282,9 @@ def compute_unique_clean_ids(strings):
     return idmap
 
 
-def escape_string(string):
-    """Escape quotes and backslashes in payee and narration.
-
-    Args:
-      string: Any string.
-    Returns.
-      The input string, with offending characters replaced.
-    """
```

Comment:
Why are you removing most of the docstring?

Reply:
it's from your suggestion https://github.com/beancount/beancount/pull/870#pullrequestreview-2438495445

Reply:
Sorry, my bad. I thought that it was evident that the suggestion applied only to the marked lines. Please fix this. Now I understand why most people find it tolerable to interact with LLMs... They apply the same level of critical thinking.

Reply:
It's also evident that and docstring is end with `"""` and rest of docstring doesn't provide any extra information

Reply:
There is a reason if the docstring had that format. Please restore the truncated part.